src/model/image_info.js

```diff
@@ -321,6 +321,17 @@ export default class ImageInfo {
      * @memberof ImageInfo
      */
     requestImgRDef(callback = null) {
+        if (callback === null)
+            callback = (rdef) => {
+                if (rdef === null || typeof rdef.c !== 'string') return;
```

Comment:
Checks are done server-side to see if the images are "compatible"
see https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/server/src/ome/logic/RenderingSettingsImpl.java#L1090